### ISSUE TITLE
When there's a syntax error, print the comment used

### ISSUE THREAD

**Issue by Wilfred** [python/mypy#4103]
Given the Python code:

```python
def big_function(
        x1, # type: Optional[int]
        x2, # type: Optional[int]
        x3, # type: Optional[int]
        x4, # type: Optional[int]
        x5, # type: Optional[int]]
        x6, # type: Optional[int]
        x7, # type: Optional[int]
):
    # type: (...) -> int
    return 123
```

Sometimes it's difficult to spot which comment has a syntax error. It would be nicer if mypy could print the comment it's looking at, to make it more obvious.

```
$ mypy --py2 bar.py
bar.py:1: error: syntax error in type comment
```

**Comment by JelleZijlstra** [comment]
I agree, but I think this should be reported upstream to https://github.com/python/typed_ast.

**Comment by Wilfred** [comment]
It looks to me like the comment parsing happens in mypy: https://github.com/python/mypy/blob/master/mypy/fastparse2.py#L310-L312 . Presumably `n.type_comment` has the comment we're interested in?

**Comment by ilevkivskyi** [comment]
Yes, I think it is possible for Python 2 to do this in mypy without changes to ``typed_ast``. On the contrary Python 3 type annotations are parsed by Python parser and should be syntactically correct.
